src/AuthContext.tsx

```diff
@@ -130,6 +130,7 @@ export const AuthProvider = ({ authConfig, children }: IAuthProvider) => {
         setRefreshTokenExpire(epochAtSecondsFromNow(refreshTokenExpiresIn))
       }
     }
+    setError(null);
```

Comment:
The function `handleTokenResponse` gets called when the access token is fetched for the first time, and during the refresh token flow. Hence, this seems the appropriate place to reset the error state.